indico/htdocs/sass/partials/_inputs.scss

```diff
@@ -458,3 +458,14 @@ dd > .i-has-action {
         padding: 0.25em 0;
     }
 }
+
+.text-holder {
```

Comment:
I think this is more like a widget. What about moving it to `/sass/widget/<something>`?